src/java.desktop/share/classes/sun/print/RasterPrinterJob.java

```diff
@@ -2418,12 +2419,12 @@ protected int printPage(Pageable document, int pageIndex)
                          * We also need to translate by the adjusted amount
                          * so that printing appears in the correct place.
                          */
-                        int bandX = deviceLeft - deviceAddressableX;
+                        int bandX = deviceLeft;
                         if (bandX < 0) {
                             bandGraphics.translate(bandX/xScale,0);
```

Comment:
Did we test this path where another translate is being done on the same object after it is done in l2399?